.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
```

Comment:
Copy-pasting only a subsection of the [Github examples](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) makes this doc a little confusing. I think it would be clearer if the entire file read like this:
```
# The following list of usernames are owners of this project 
# and will tagged as reviewers when you create a PR:

*       @VladimirKravtsov @akirillov @rpalaznik @farhan5900 @samvantran @alembiewski

# To add more rules, see: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners

Reply:
> * sometimes it is sufficient to request a review from just a single person when the change is related to some specific code area developed by that person

 To cover that case,  it is possible to add to the file a more specific mapping like ’/kudo-spark @akirillov’. I think when responsibilities for the operator pieces became persistent across team members, we can easily add such mappings. 

> * spamming the whole dev team every time a PR is created can lead to a situation when the actual requests are ignored because "anyway, the whole team received the same request."

I walked through across the last 10 - 15 Rs in this repo, and 90% of them have the entire pillar team in reviewers. It means having current CODEOWNERS, in most cases, will no need to adjust the reviewers list.
 
> TL;DR requesting a review on per-PR basis doesn't seem to be an issue and works for all other repos so far.

Yes, it is. 

the codeowners configuration maybe bring some additional convenience into the development process, but perhaps not if so, we can easily remove it.